manifests/stage-ofed-driver/0001_namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
Please, add license header for all new non-empty files

Reply:
None of the manifests have license headers. Why we want to add them now?

Reply:
actually they do. (some of them) but this is not needed for new files. (internal process that changed..)

however, i see a greater problem.
This namespace needs to be created when we deploy operator. i.e when deploying with:
- make deploy
- helm chart
- olm

stage OFED driver can be skipped in general, moreover this is not the first stage that is applied.